src/commands/schema/diff.ts

```diff
@@ -1,9 +1,14 @@
 import SchemaCommand from "../../lib/schema-command";
-import { colorParam, hasColor } from "../../lib/color";
+import { bold, colorParam, hasColor, reset } from "../../lib/color";
+import { Flags } from "@oclif/core";
 
 export default class DiffSchemaCommand extends SchemaCommand {
   static flags = {
     ...SchemaCommand.flags,
+    active: Flags.boolean({
+      description: "Compare the local schema to the active schema.",
+      default: false,
+    }),
   };
 
   static description = "Print the diff between local and remote schema.";
```

Comment:
thoughts on clarifying like:
`Compare the local schema to the active schema instead of the staged schema.`

and

`Print the diff between the local schema and the staged remote schema.`